### ISSUE TITLE
Format dates according to user's locale

### ISSUE THREAD

**Issue by thatjpk** [nextcloud/passman#303]
### Steps to reproduce
1. Create a vault
2. Log out of vault to view the vault list

![screenshot](https://cloud.githubusercontent.com/assets/1297471/25883877/15565988-351c-11e7-9dda-65b8e68f60b8.PNG)

### Expected behaviour
The created and last-accessed date should be formatted according to the user's locale.

### Actual behaviour
The created and last-accessed date are always formatted as `dd-mm-yyyy`

### Configuration
**Operating system**: Windows 10 

**Browser**: Google Chrome 58.0.3029.96

**Extensions that might cause interference**: Files, Activity, Gallery, Passwords, Audio Player

**Passman version**: 2.1.2

**Operating system**: Ubuntu 16.04.2

**Web server:** Apache 2.4.18

**Database:** PostgreSQL 9.5

**PHP version:** 7.0.15
      
**cloud server:** Nextcloud

**cloud version:**  11.0.3

**Comment by thatjpk** [comment]
Looks like [this](https://github.com/nextcloud/passman/blob/375c4e341b7798de560e134aa3bd62181fb96644/templates/views/vaults.html#L15-L18) is where that particular formatting happens?  A quick search for "dd-MM-yyyy" shows this approach is used in a few other templates.

**Comment by brantje** [comment]
Yea, that's the place where formatting happens.

**Comment by brantje** [comment]
@thatjpk Can you login to Nextcloud / ownCloud, open the console and paste the following:
`moment.localeData().longDateFormat('L')` 
This gets the formatted date in my case: `"DD/MM/YYYY"` yours might be: `"MM/DD/YYYY"`
If you can confirm that is the correct output i can implement it.

**Comment by thatjpk** [comment]
@brantje Yep, that returns `"MM/DD/YYYY"` for me and looks right for my locale
